soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1603,82 +1603,86 @@ void Randomizer::ParseRandomizerSettingsFile(const char* spoilerFileName) {
                         break;
                     case RSK_HINT_CLARITY:
-                        if(it.value() == "Obscure") {
-                            gSaveContext.randoSettings[index].value = 0;            
-                        } else if(it.value() == "Ambiguous") {
+                        if (it.value() == "Obscure") {
+                            gSaveContext.randoSettings[index].value = 0;
+                        } else if (it.value() == "Ambiguous") {
                             gSaveContext.randoSettings[index].value = 1;
-                        } else if(it.value() == "Clear") {
+                        } else if (it.value() == "Clear") {
                             gSaveContext.randoSettings[index].value = 2;
                         }
                         break;
                     case RSK_HINT_DISTRIBUTION:
-                        if(it.value() == "Useless") {
-                            gSaveContext.randoSettings[index].value = 0;            
-                        } else if(it.value() == "Balanced") {
+                        if (it.value() == "Useless") {
+                            gSaveContext.randoSettings[index].value = 0;
+                        } else if (it.value() == "Balanced") {
                             gSaveContext.randoSettings[index].value = 1;
-                        } else if(it.value() == "Strong") {
+                        } else if (it.value() == "Strong") {
                             gSaveContext.randoSettings[index].value = 2;
-                        } else if(it.value() == "Very Strong") {
+                        } else if (it.value() == "Very Strong") {
                             gSaveContext.randoSettings[index].value = 3;
                         }
                         break;
                     case RSK_GANONS_BOSS_KEY:
-                        if(it.value() == "Start With") {
-                            gSaveContext.randoSettings[index].value = 0;            
-                        } else if(it.value() == "Vanilla") {
+                        if (it.value() == "Start With") {
+                            gSaveContext.randoSettings[index].value = 0;
+                        } else if (it.value() == "Vanilla") {
                             gSaveContext.randoSettings[index].value = 1;
-                        } else if(it.value() == "Own Dungeon") {
+                        } else if (it.value() == "Own Dungeon") {
                             gSaveContext.randoSettings[index].value = 2;
                         }
                         break;
                     case RSK_SKIP_CHILD_ZELDA:
-                        gSaveContext.randoSettings[index].value = it.value();
+                        if (it.value() == "Off") {
+                            gSaveContext.randoSettings[index].value = 0;
+                        } else if (it.value() == "On") {
+                            gSaveContext.randoSettings[index].value = 1;
+                        }
```

Comment:
revert rest of file except this part? seems like a lot of style was changed in the rest of the file

Reply:
might actually make sense to revert all of the changes to this file since the bool vs. string issue wasn't actually an issue

Reply:
formatting should be applied once with clang-format and kept that way going forward . it doesn't belong in this PR

Reply:
fair enough!